### PR TITLE
Documented the "autoescape" TwigBundle config option

### PR DESCRIPTION
**Pull request by javiereguiluz**
This option was missing after the recent doc update.

### CODE REVIEW

**Comment by stof** [comment]
I would add a note saying that disabling the autoescaping globally could open XSS holes as all bundles in the ecosystem rely on auto-escaping rather than explicitly escaping everything. So using ``false`` here is dangerous.

**Comment by javiereguiluz** [comment]
@stof thanks for your insightful comment. That's definitely something we need to mention. I've updated the PR. Thanks!

File: reference/configuration/twig.rst [hunk 1]
```diff
@@ -112,6 +112,28 @@ If ``true``, whenever a template is rendered, Symfony checks first if its source
 code has changed since it was compiled. If it has changed, the template is
 compiled again automatically.
 
+autoescape
+~~~~~~~~~~
+
+**type**: ``boolean`` or ``string`` **default**: ``'filename'``
+
+If set to ``true``, all template contents are escaped for HTML. If set to
+``false``, no escaping is applied automatically (you can still escape each
```
Comment (WouterJ) [inline reference/configuration/twig.rst]: "If set to `false`, automatic escaping is disabled (you can still [...]" ?

File: reference/configuration/twig.rst [hunk 2]
```diff
@@ -112,6 +112,28 @@ If ``true``, whenever a template is rendered, Symfony checks first if its source
...
+content individually in the templates).
+
+.. caution::
+
+    Setting this option to ``false`` is dangerous and it will make your
+    application vulnerable to XSS exploits because most third-party bundles
+    assume that auto-escaping is enabled and they don't escape contents
+    themselves.
+
+If set to a string, template contents are escaped using the strategy with that
```
Comment (WouterJ) [inline reference/configuration/twig.rst]: I think I'm missing "the" before "template contents", not 100% sure though but it feels more correct

File: reference/configuration/twig.rst [hunk 3]
```diff
@@ -112,6 +112,28 @@ If ``true``, whenever a template is rendered, Symfony checks first if its source
...
+name. Allowed values are ``html``, ``js``, ``css``, ``url`` and ``html_attr``.
```
Comment (WouterJ) [inline reference/configuration/twig.rst]: please also add the default value in the allowed values list

File: reference/configuration/twig.rst [hunk 4]
```diff
@@ -112,6 +112,28 @@ If ``true``, whenever a template is rendered, Symfony checks first if its source
...
+A special strategy called ``filename`` is used by default and it escapes contents
+according to the filename extension (it uses ``html`` for ``*.html.twig``
```
Comment (WouterJ) [inline reference/configuration/twig.rst]: "[...] (e.g. it uses [...])"

File: reference/configuration/twig.rst [hunk 5]
```diff
@@ -112,6 +112,28 @@ If ``true``, whenever a template is rendered, Symfony checks first if its source
...
+templates, ``js`` for ``*.js.html`` templates, etc.)
```
Comment (WouterJ) [inline reference/configuration/twig.rst]: what about adding a ref to the next option, allowing people to use a custom strategy?

**Comment by javiereguiluz** [comment]
@WouterJ thanks for the review. I've fixed everything.

**Comment by xabbuh** [comment]
👍

**Comment by WouterJ** [comment]
Thanks @javiereguiluz! It looks great. I've made some very minor changes in c7e2145

**Pull request closed without merging**